### PR TITLE
EventGrid: Regenerated Python SDK based on latest API version.

### PR DESCRIPTION
**Pull request by kalyanaj**
Regenerated EventGrid Python SDK based on latest API version. Corresponding OpenAPI spec pull request is at: https://github.com/Azure/azure-rest-api-specs/pull/2239.

### CODE REVIEW

File: azure-mgmt-eventgrid/azure/mgmt/eventgrid/models/__init__.py [hunk 1]
```diff
@@ -33,7 +33,6 @@
 from .topic_type_info_paged import TopicTypeInfoPaged
 from .event_grid_management_client_enums import (
     EventSubscriptionProvisioningState,
-    OperationOrigin,
```
Comment (lmazuel) [inline azure-mgmt-eventgrid/azure/mgmt/eventgrid/models/__init__.py]: I don't see removal of this file in the PR, you probably regenerated in place. Could you erase your azure-mgmt-eventgrid/azure/mgmt/eventgrid/ folder and regenerate?

Also, please add removal of this enum in the ChangeLog

Comment (lmazuel) [reply azure-mgmt-eventgrid/azure/mgmt/eventgrid/models/__init__.py]: Nope, that's an enum, no file removal. But still would appreciate if you can add that in the ChangeLog :)

Comment (kalyanaj) [reply azure-mgmt-eventgrid/azure/mgmt/eventgrid/models/__init__.py]: Done, thanks!

**Comment by lmazuel** [comment]
@kalyanaj Travis is struck today, probably this means no release for the day (let's be pessimist). Do want a release asap?

**Pull request merged**